src/webgl.js

```diff
@@ -239,7 +239,7 @@ function init_conv1_1_program(gl) {
   const operations = [];
 
   for (let i = 0; i < 5; i++) {
-    coords.push(`vec2 coords_${i} = vec2(outX / inWidth, (outY + ${i}.0) / inHeight);`);
+    coords.push(`vec2 coords_${i} = vec2(outX * inWidthInverse, (outY + ${i}.0) * inHeightInverse);`);
```

Comment:
Nice! I wonder if it'd be faster to pack `inInverse` as a `vec2` and work from there? I'm not sure because of the modification of `outY` in this phrase, but if there's a nicer way to represent that, then it might work.